cmds/core/mv/mv.go

```
@@ -23,19 +23,18 @@ import (
 var (
 	update    = flag.Bool("u", false, "move only when the SOURCE file is newer than the destination file or when the destination file is missing")
 	noClobber = flag.Bool("n", false, "do not overwrite an existing file")
-	osExit    = os.Exit
+)
+var (
+	errOsExit error = fmt.Errorf("os.Exit(1)")
 )
 
-func usage() {
+func usage() error {
 	fmt.Fprintf(os.Stderr, "Usage: %s [ARGS] source target\n", os.Args[0])
 	fmt.Fprintf(os.Stderr, "       %s [ARGS] source ... directory\n", os.Args[0])
 	flag.PrintDefaults()
-	osExit(1)
+	return errOsExit
```

usage does not need to return an error. See comment below.

---

maybe below was wrong. it appears above :D